charts/clickhouse/templates/clickhouse-instance/scripts-configmap.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "clickhouse.fullname" . }}-scripts
+  namespace: {{ .Values.namespace | default .Release.Namespace }}
+  labels:
+    {{- include "clickhouse.labels" . | nindent 4 }}
+data:
+  wait-until-ready.sh: |
+    #!/bin/bash
+    
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)